### ISSUE TITLE
There is no proper document for SAML2 IdP Initiated Single Logout with WSO2 Identity Server

### ISSUE THREAD

**Issue by NilukaSripalim** [wso2/docs-is#173]

**Comment by ayshsandu** [comment]
https://is.docs.wso2.com/en/5.9.0/learn/configuring-saml2-web-single-sign-on/#manual-configuration document explain how to enable Idp initiated SLO.
Is it something different from what you expect from $subject.

**Comment by NilukaSripalim** [comment]
Yes, That document mentions that we are supporting "IdP Initiated SLO", but  I was unable to see any details which describe the "IdP initiated SLO request" and parameters that are used with IdP initiated SLO request.
Ex: there is no example or brief description of how we can use parameters, "spEntityID" , "returnTo"
